content/en/templates/taxonomy-templates.md

```diff
@@ -126,9 +126,13 @@ Taxonomies can be ordered by either alphabetical key or by the number of content
 
 ```
 <ul>
-  {{ $data := .Data }}
+  {{ $type := .Type }}
   {{ range $key, $value := .Data.Terms.Alphabetical }}
-  <li><a href="{{ $.Site.LanguagePrefix }}/{{ $data.Plural }}/{{ $value.Name | urlize }}"> {{ $value.Name }} </a> {{ $value.Count }} </li>
+  {{ $name := .Name }}
+  {{ $count := .Count }}
+  {{ with $.Site.GetPage "taxonomy" $type $name }}
```

Comment:
`.Site.GetPage` will accept only 2 args in the upcoming Hugo release. So this needs to be refactored to:

```
{{ with $.Site.GetPage "taxonomy" (printf "%s/%s" $type $name) }}
```

[Examples from my site theme][1]

[1]: https://gitlab.com/kaushalmodi/hugo-theme-refined/commit/7c03871829009081c71499afb9dee6aaaf65ebed

Reply:
I would appreciate if this could wait. I have rewritten this in the /hugo repo, but it will not be merge into this repo until release date (Monday) -- and I would hate having to do manual merge stuff ...

Reply:
Makes sense. 

@shaform How above you rebase this PR after Monday?

Reply:
@kaushalmodi Sure.

Reply:
@kaushalmodi I've rebased this PR and made it to only use 2 args.